translator/translate/otel/processor/metricstransformprocessor/appsignals_runtime_config.yaml

```diff
@@ -277,3 +277,121 @@ transforms:
       - action: add_label
         new_label: Telemetry.Source
         new_value: RuntimeMetric
+  - include: process.runtime.dotnet.gc.collections.count
```

Comment:
I assume these rules will just be a no-op if runtime_metrics is disabled and these metrics arent sent at all in that case?

Reply:
The runtime_metrics control is at SDK side. The CloudWatch agent stands by to handle runtime metrics whenever the metrics come. If no metrics is received, the rules won't be applied.